### PR TITLE
Feature/tech debt removal

### PR DESCRIPTION
**Pull request by Tiewhan**
<h1>Overview</h1>
This PR deals with removing tech debt in the main project.

<h2>Type of Debt Removed/Fixed</h2>

- Interface colours now use colours defined in asset catalog instead
- Views with drop shadows and rounded corners now use the respective extensions methods that were applied in an earlier branch
- File Renaming/Moving
- Some code was moved into the main thread as it was a UI operation
- Some code was moved into a function to make it more readable

<h2>Important Note about UI Tests</h2>
The current UI tests have been disabled due to the massive amount of changes being done. These tests will done redone on 28th of April alongside new unit tests that cover the entire application.

### CODE REVIEW

**Review by AkuaQ — APPROVED** [review]
You already know that the comments cannot stay🙂. Apart from that, looks good.

File: Feisty/FriendsView/FriendsViewController.swift [hunk 1]
```diff
@@ -62,10 +62,14 @@ class FriendsViewController: UITableViewController {
   
   func createAndShowAlert(message: String, handler: @escaping((UIAlertAction) -> Void)) {
     
-    let alertController = UIAlertController(title: "Error", message: "", preferredStyle: .alert)
-    alertController.message = message
-    alertController.addAction(UIAlertAction(title: "Dismiss", style: .default, handler: handler))
-    self.present(alertController, animated: true, completion: nil)
+    DispatchQueue.main.async {
+      
+      let alertController = UIAlertController(title: "Error", message: "", preferredStyle: .alert)
```
Comment (Zamo22) [inline Feisty/FriendsView/FriendsViewController.swift:67]: Just a suggestion, if you plan on having error alerts in the future, would be a good idea to make a generic view extension that takes an error message and displays it. You can customize with taking closures as well if you want to do extensions for other alerts. Not necessary to implement if you're not using it in a lot of places though

Comment (Tiewhan) [reply Feisty/FriendsView/FriendsViewController.swift:67]: That is a fantastic idea for the error propagation that is planned. Was thinking how I would do it and that is great.

File: FeistyUITests/GameViewTests/GameViewUITests.swift [hunk 2]
```diff
@@ -14,42 +14,42 @@ class GameViewUITests: GameViewTestCase {
...
-      
-      self?.application.navigationBars.buttons["Games"].tap()
-      XCTAssert(self?.application.staticTexts["Games In Store"].exists ?? false)
+//    application.launch()
+//
+//    loginUserWithDetails()
+//
+//    let gameNameLabel = application.staticTexts["test2"]
+//    let exists = NSPredicate(format: "exists == 1")
+//
```
Comment (Zamo22) [inline FeistyUITests/GameViewTests/GameViewUITests.swift]: Should probably remove this

**Review by Zamo22 — APPROVED** [review]
No issues but would remove commented UI tests. Approving in the meantime

**Pull request merged**